pilot/PilotUserService.swift

```diff
@@ -36,14 +36,17 @@ class PilotUserService: NSObject {
                       completion: @escaping (PilotUser) -> Void,
                       failure: @escaping (HTTPStatusCode) -> Void) {
         
+        let basicCredentials = "\(PilotConfiguration.Thunder.userKey):\(PilotConfiguration.Thunder.userSecret)"
+            .data(using: String.Encoding.utf8)!.base64EncodedString(options: [])
+        
```

Comment:
This may need to more thought. The problem is we don't actually have a PilotUser for the log in request because that's normally returned after a log in request so I have to build the URL and all it's headers & parameters manually for Alamofire to make the request.